src/app/user/login/index.njk

```diff
@@ -44,21 +47,21 @@
 	<div class="govuk-grid-column-two-thirds">
 		<form action="login" autocomplete="off" method="POST">
 			<h1 class="govuk-heading-xl">{{ __('button_sign_in') }}</h1>
-			<div class="govuk-form-group {{ m.form_group_class('Username', errors) }}">
```

Comment:
Renamed "Username" to "username" as it was bugging me. Post controller then sets it as usrname when it's doing it's thing.